.chronus/changes/diag-path-relative-2024-9-21-16-52-22.md

```diff
@@ -0,0 +1,8 @@
+---
+# Change versionKind to one of: internal, fix, dependencies, feature, deprecation, breaking
+changeKind: feature
+packages:
+  - "@typespec/compiler"
+---
+
+CLI logs diagnostic source path relative to the CWD.
```

Comment:
```suggestion
CLI logs diagnostic source path relative to the CWD.
```

right now it is relative to cwd which I think is what make sense for usability(so you can tsp compile `./path/to/sub-spec` and have links still work. The alternative would be do it relative to the project root.

TypeScript has an issue about doing the opposite change https://github.com/microsoft/TypeScript/issues/36221, rust also had but closed due to lack of intrest.

So it seems like we might want an option as well to change(Could also wait and see if people don't prefer the current flow)